filebeat/docs/modules/salesforce.asciidoc

```diff
@@ -253,6 +254,12 @@ The password used as part of the authentication flow. It is required for authent
 
 The URL of the Saleforce instance.
 
+*`var.interval`*::
+
+Period of fetching logs, i.e. 1s/1m/1h. 
```

Comment:
What will be the impact if user selects a lower limit like 1m, as given above?

Reply:
> What will be the impact if user selects a lower limit like 1m, as given above?

If the user provides a lower limit like 1m, there won't be any issue unless and until the Salesforce REST API rate limit exceeds. I have mentioned the `Note` in which users can find the Salesforce REST API rate limit from the official doc.